.github/workflows/regenerate-cmdline-ref.yml

```
@@ -88,29 +155,31 @@ jobs:
           body: |
-            ${{ 
-              steps.build.conclusion == 'failure'
-              && format('❌ Slang build failed. Please check the [workflow run](https://github.com/{0}/actions/runs/{1})', github.repository, github.run_id)
-              || (steps.regen.conclusion == 'failure' 
-                  && format('❌ Command line reference generation failed. Please check the [workflow run](https://github.com/{0}/actions/runs/{1})', github.repository, github.run_id)
-                  || (steps.create-pr.conclusion == 'failure' 
-                      && format('❌ Failed to create regenerate command line reference pull request. Please check the [workflow run](https://github.com/{0}/actions/runs/{1})', github.repository, github.run_id)
-                      || format('🌈 Regenerated command line reference, please merge the changes from [this PR]({0})', steps.create-pr.outputs.pull-request-url)))
+            ${{
+              needs.generate_cmdline_ref.result != 'success'
+              && format('❌ Command line reference generation failed. Please check the [workflow run](https://github.com/{0}/actions/runs/{1})', github.repository, github.run_id)
```

actionlint (1.7.12)

[error] calling function "failure" is not allowed here. "failure" is only available in "jobs.<job_id>.if", "jobs.<job_id>.steps.if". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)